prisma/migrations/20230402192938_/migration.sql

```diff
@@ -59,6 +59,8 @@ CREATE TABLE "Account" (
     "refresh_token" TEXT,
     "access_token" TEXT,
     "expires_at" INTEGER,
+    "expires_in" INTEGER,
+    "ext_expires_in" INTEGER,
     "token_type" TEXT,
```

Comment:
are these really neccessary?

Reply:
> are these really neccessary?

I can try without but seems necessary according to documentation next auth.